src/interfaces/coral_web/src/components/Shared/Logo.tsx

```diff
@@ -2,64 +2,88 @@ import cx from 'classnames';
+          className={cx({
+            'fill-quartz-500': style === 'default',
+            'fill-primary-200': style === 'coral',
+            'fill-marble-400': style === 'grayscale',
+            'dark:fill-marble-100': darkModeEnabled,
+          })}
+        />
+        <path
+          d="M2.74588 10.1523H2.74583C1.22935 10.1523 0 11.3817 0 12.8982V13.2538C0 14.7703 1.22935 15.9996 2.74583 15.9996H2.74588C4.26235 15.9996 5.4917 14.7703 5.4917 13.2538V12.8982C5.4917 11.3817 4.26235 10.1523 2.74588 10.1523Z"
+          className={cx({
+            'fill-primary-500': style === 'default',
+            'fill-primary-200': style === 'coral',
+            'fill-marble-400': style === 'grayscale',
+            'dark:fill-marble-100': darkModeEnabled,
+          })}
+        />
+      </svg>
+    )
+  }
+
+
+  const showCustomLogo = () => {
+    /** 
+     * @description Modify this section to render a custom logo or text based on specific design guidelines.
+     */
+    
+    return (
+      <img
+        src="/images/logo.png"
+        alt="Logo"
+        className={cx('h-full', { 'w-24': includeBrandName, 'w-4': !includeBrandName }, className)}
       />
-    )}
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M5.18375 9.52649C5.6144 9.52649 6.47106 9.50284 7.65517 9.01533C9.03504 8.44723 11.7804 7.41592 13.7607 6.35661C15.1458 5.6157 15.7529 4.63579 15.7529 3.31618C15.7529 1.48471 14.2682 0 12.4368 0H4.76324C2.13257 0 0 2.13257 0 4.76324C0 7.39392 1.99672 9.52649 5.18375 9.52649Z"
-      className={cx({
-        'fill-green-700': style === 'default',
-        'fill-primary-200': style === 'coral',
-        'fill-marble-400': style === 'grayscale',
-        'dark:fill-marble-100': darkModeEnabled,
-      })}
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M6.48047 12.8071C6.48047 11.5176 7.25678 10.3549 8.44779 9.86059L10.8644 8.85769C13.3087 7.84323 15.9991 9.63954 15.9991 12.2861C15.9991 14.3365 14.3366 15.9985 12.2862 15.998L9.6698 15.9973C7.90823 15.9968 6.48047 14.5686 6.48047 12.8071Z"
-      className={cx({
-        'fill-quartz-500': style === 'default',
-        'fill-primary-200': style === 'coral',
-        'fill-marble-400': style === 'grayscale',
-        'dark:fill-marble-100': darkModeEnabled,
-      })}
-    />
-    <path
-      d="M2.74588 10.1523H2.74583C1.22935 10.1523 0 11.3817 0 12.8982V13.2538C0 14.7703 1.22935 15.9996 2.74583 15.9996H2.74588C4.26235 15.9996 5.4917 14.7703 5.4917 13.2538V12.8982C5.4917 11.3817 4.26235 10.1523 2.74588 10.1523Z"
-      className={cx({
-        'fill-primary-500': style === 'default',
-        'fill-primary-200': style === 'coral',
-        'fill-marble-400': style === 'grayscale',
-        'dark:fill-marble-100': darkModeEnabled,
-      })}
-    />
-  </svg>
-);
+    );
+  }
+
+
+  return hasCustomLogo === 'true' ? showCustomLogo() : showDefaultLogo();
```

Comment:
i think instead of functions we can just do the following:

```suggestion
  if (hasCustomLogo === 'true') {
    // Modify this section to render a custom logo or text based on specific design guidelines.
    return (
      <img
        src="/images/logo.png"
        alt="Logo"
        className={cx('h-full', className)}
      />
    );
  }
  
  return (....); // original svg goes here
```

I also removed the part about brand name for the custom logo since it shouldn't matter for the custom logo